jobs/build/drop_advisories/Jenkinsfile

```diff
@@ -40,15 +45,22 @@ node {
     advisory_list = commonlib.parseList(params.ADVISORIES)
     
     for(adv in advisory_list) {
+        if (!params.COMMENT) {
+            comment = "This bug will be dropped from ${adv} since the advisory will be dropped."
```

Comment:
Would like to set the default comment content as the default value of `COMMENT` parameter.